src/components/Card/Card.vue

```diff
@@ -8,24 +12,7 @@
     <p>Humidity: {{ humidity }}%</p>
     <h3>Air Quality</h3>
     <p>{{ aqi }} AQI</p>
-    <div v-if="aqi <= 50">
-      🟢 Good
-    </div>
-    <div v-else-if="aqi >= 51 && aqi <= 100">
-      🟡 Moderate
-    </div>
-    <div v-else-if="aqi >= 101 && aqi <= 150">
-      🟠 Unhealthy for Sensitive Groups
-    </div>
-    <div v-else-if="aqi >= 151 && aqi <= 200">
-      🔴 Unhealthy
-    </div>
-    <div v-else-if="aqi >= 201 && aqi <= 300">
-      🟣 Very Unhealthy
-    </div>
-    <div v-else-if="aqi >= 301">
-      💀 Hazardous
-    </div>
+    <p>{{ determineAqiMessage() }}</p>
```

Comment:
We need to figure out what we want to display for more descriptive / helpful information to inform a user of how the reported AQI will impact their choice for engaging in activities